manim/utils/hashing.py

```diff
@@ -46,7 +46,7 @@ def default(self, obj):
             )
         elif isinstance(obj, np.ndarray):
             if obj.size > 1000:
-                return f"Np array too big {obj.size}"
+                obj.resize((100, 100))
```

Comment:
This might be a bit confusing when debugging if we don't know that the array has been truncated.

Maybe return the `repr` of the array with the string `truncated` at the beginning or end of the `repr`?

Reply:
Good point. I will work on that